### PR TITLE
Schedule Instance Policy - WIP

### PR DESCRIPTION
**Pull request by cdwilhelm**
### Description
New Policy for Scheduling instances

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [x] All tests pass.
- [ ] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,8 +11,9 @@ Please contact sales@rightscale.com to learn more.
 - [AWS Reserved Instances Utilization](./cost/aws/reserved_instances/utilization/)
 - [AWS Reserved Instance Reservation Coverage](./cost/aws/reserved_instances/coverage/)
 - [Downsize Instances](./cost/downsize_instance/)
-- [Unattached Volumes Policy](./cost/volumes/unattached_volumes/)
```
Comment (bharatrightscale) [inline README.md]: @cdwilhelm this doesn't seem right. The path is correct as before.

File: cost/schedule_instances/schedule_instances.pt [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+See [README](https://github.com/rightscale/policy_templates/cost/schedule_instances) for more details"
+long_description "Version 1.0"
+severity "medium"
+category "Cost"
+
+
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses"
```
Comment (bharatrightscale) [inline cost/schedule_instances/schedule_instances.pt]: I think we can use the same label name as other policies. I went ahead and made them consistent a few weeks ago if it helps.

File: README.md (hunk 1 above)
Comment (cdwilhelm) [reply README.md]: not sure why it changed,  maybe it was part of a merge.  anyway it's fixed now.

Comment (cdwilhelm) [reply README.md]: I fixed this

File: cost/schedule_instances/schedule_instances.pt (hunk 2 above)
Comment (cdwilhelm) [reply cost/schedule_instances/schedule_instances.pt]: good catch.  I updated this.

**Comment by MitchellGerdisch** [comment]
Looks good to me.
But the policy template should be added to 
https://github.com/rightscale/policy_templates/blob/master/active-policy-list.json

**Pull request merged**